packages/lib-classifier/src/plugins/drawingTools/experimental/components/TranscriptionLine/TranscriptionLine.spec.js

```diff
@@ -227,8 +227,43 @@ describe('Components > Drawing marks > Transcription line', function () {
         }
       )
       const dragMove = wrapper.find(DragHandle).find('[x=300]').at(0).prop('dragMove')
+      
+      wrapper.find('circle[cx=300]').at(0).simulate('pointerdown')
+      expect(mark.x2).to.equal(300)
+      expect(mark.y2).to.equal(400)
+      dragMove({}, { x: 10, y: 20 })
```

Comment:
I initially thought this should be something along the lines of:
`wrapper.find('circle[cx=300]').at(0).simulate('pointermove', { x: 10, y: 20 })`
but was unable successfully implement.